apps/datahub-e2e/src/e2e/datasetDetailPage.cy.ts

```diff
@@ -417,11 +423,9 @@ describe('dataset pages', () => {
           .find('gn-ui-copy-text-button')
           .find('button')
           .first()
-          .click({ force: true })
+          .realClick()
```

Comment:
What's a `realClick`? :D

Reply:
Instead of calling a js click function, cypress fires a native click.